### PR TITLE
Pick up performance test changes

### PR DESCRIPTION
**Pull request by Sindica**

### CODE REVIEW

File: cluster/gce/manifests/etcd.manifest [hunk 1]
```diff
@@ -45,7 +45,7 @@
         "value": "{{ etcd_protocol }}://{{ hostname }}:{{ server_port }}"
       },
       { "name": "ETCD_SNAPSHOT_COUNT",
-        "value": "10000"
+        "value": "100000"
       }
```
Comment (sonyafenge) [inline cluster/gce/manifests/etcd.manifest:49]: do we really need this change? I checked community still using "10000" as value.

File: cluster/common.sh [hunk 2]
```diff
@@ -1132,10 +1132,8 @@ function registry-authentication() {
 function create-and-upload-etcd-image() {
 
   ETCD_IMAGE_REGISTRY="${ETCD_IMAGE_REGISTRY:-${CONTAINER_REGISTRY}/${PROJECT}}"
-
   # Build+push the image through makefile.
   MAKE_DIR="${KUBE_ROOT}/cluster/images/etcd"
-
```
Comment (sonyafenge) [inline cluster/common.sh:1138]: this commit already included in master.

File: cluster/gce/gci/configure-helper.sh [hunk 3]
```diff
@@ -130,7 +130,7 @@ function main() {
     start-kube-controller-manager
     start-kube-scheduler
     wait-till-apiserver-ready
-    start-workload-controller-manager
+   # start-workload-controller-manager
```
Comment (yb01) [inline cluster/gce/gci/configure-helper.sh:133]: can we use this variable to control it as the partitionserver-configure-helper.sh does ?
{ENABLE_WORKLOADCONTROLLER}

**Review by yb01 — APPROVED** [review]
/lgtm

File: cluster/gce/manifests/etcd.manifest (hunk 1 above)
Comment (Sindica) [reply cluster/gce/manifests/etcd.manifest:49]: In that case we should use 10K in arktos run. I noticed recent 1.18 test used 10K. Please check with the team.

File: cluster/gce/gci/configure-helper.sh (hunk 3 above)
Comment (Sindica) [reply cluster/gce/gci/configure-helper.sh:133]: The issue is that we need additional work to disable rs controller and deployment controller in KCM. This PR is trying to synchronize perf test and master, would rather leave enhancement in separate changes.

File: cluster/common.sh (hunk 2 above)
Comment (Sindica) [reply cluster/common.sh:1138]: Can you specify the commit comment?

File: cluster/gce/gci/configure-helper.sh (hunk 3 above)
Comment (sonyafenge) [reply cluster/gce/gci/configure-helper.sh:133]: we do have this parameter, but in this time, it only works for partition servers and it doesn't work for master server. I agree with Ying, for now, we can keep this way.

File: cluster/common.sh (hunk 2 above)
Comment (sonyafenge) [reply cluster/common.sh:1138]: included in this PR: https://github.com/CentaurusInfra/arktos/pull/796

**Pull request merged**